### PR TITLE
[MM-10974] Remove Character Limit in Reset Email Modal

### PR DESCRIPTION
**Pull request by R-Wang97**
#### Summary
Remove the character limit that currently exists in the reset email modal under system console.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-10974

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed

### CODE REVIEW

File: components/admin_console/reset_email_modal/reset_email_modal.jsx [hunk 1]
```diff
@@ -121,7 +121,6 @@ export default class ResetEmailModal extends React.Component {
                                         type='email'
                                         ref='email'
                                         className='form-control'
-                                        maxLength='22'
```
Comment (lieut-data) [inline components/admin_console/reset_email_modal/reset_email_modal.jsx]: Should we copy the paradigm in [signup_email.jsx](https://github.com/mattermost/mattermost-webapp/blob/master/components/signup/signup_email/signup_email.jsx#L351) and just set this to `128`?

Comment (R-Wang97) [reply components/admin_console/reset_email_modal/reset_email_modal.jsx]: Will set it to `128` then.

Comment (saturninoabril) [reply components/admin_console/reset_email_modal/reset_email_modal.jsx]: Cool, thanks!
(Don't know why did I mention admin console link above.  I thought it's different from this one.)

**Comment by jasonblais** [comment]
@lieut-data Is this good to merge now?

@R-Wang97 Looks like this needs a rebase. Thanks for working on this! :)

**Comment by jasonblais** [comment]
Thanks! This can be merged for v5.1 @cpanato

**Comment by cpanato** [comment]
@jasonblais that means we need to wait for the branch? or we can merge to master now?

**Comment by jasonblais** [comment]
@cpanato We can merge to `master` given we haven't cut the 5.1 release branch yet

**Comment by cpanato** [comment]
Ok

**Pull request merged**